KD63799/deadlinedrive
# Installation of dependencies

# Description:
This pull request integrates Tailwind CSS into our React project along with other necessary dependencies for the development of our final project.

## Changes Made:

### Installation of Tailwind CSS:
- Added Tailwind CSS as a development dependency.
- Configured the `tailwind.config.js` file to include our application's paths.
- Included Tailwind CSS directives (`@tailwind base;`, `@tailwind components;`, `@tailwind utilities;`) in the `index.css` file.

### Project Structure Setup:
- Created the `src/assets`, `src/pages`, and `src/components` directories.
- Organized files into appropriate directories for better code management.

### Configuration of React Router:
- Installed `react-router-dom` to manage navigation between different pages of our application.
- Configured routes in `App.jsx` using `BrowserRouter`, `Routes`, and `Route`.

### Example Component with Tailwind CSS:
- Created a `Home` component with basic Tailwind CSS usage to demonstrate style application.

## Testing Instructions:
1. Clone the repository and switch to this branch.
2. Run `npm install` to install all dependencies.
3. Start the application using `npm run dev` (or the appropriate command for your setup).
4. Navigate to different pages to verify that the React Router configuration works.
5. Verify that Tailwind CSS styles are correctly applied by inspecting the `Home` component.

## Added Dependencies:
- `tailwindcss`
- `postcss`
- `autoprefixer`
- `react-router-dom`

Please review and approve this pull request so we can proceed with the development of our project using Tailwind CSS and the configured dependencies.
